Mathlib/Order/Lattice.lean

```diff
@@ -127,6 +127,10 @@ theorem sup_le_iff : a ⊔ b ≤ c ↔ a ≤ c ∧ b ≤ c :=
   ⟨fun h : a ⊔ b ≤ c => ⟨le_trans le_sup_left h, le_trans le_sup_right h⟩,
    fun ⟨h₁, h₂⟩ => sup_le h₁ h₂⟩
 
+@[simp]
+theorem sup_self : a ⊔ a = a := by
```

Comment:
I believe this already exists as `sup_idem` (and similarly for the inf lemma below).

Reply:
Ah, thanks!